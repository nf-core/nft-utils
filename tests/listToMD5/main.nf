workflow TEST {
    main:
    def fill = "This needs to be filled in"
    def lines = [
        "This is a line",
        32,
        453.2,
        """
        Wow, a multiline string
        """,
        "${fill}"
    ]

    emit:
    lines_out = lines
}