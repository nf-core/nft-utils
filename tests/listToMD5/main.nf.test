nextflow_workflow {

    name "Test Workflow TEST"
    script "./main.nf"
    workflow "TEST"

    tag "listToMD5"

    test("Should run without failures") {

        then {
            assert workflow.success
            assert snapshot(workflow.out.lines_out.collect { listToMD5(it) }).match()
        }

    }

}
