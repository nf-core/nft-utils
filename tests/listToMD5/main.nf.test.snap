{
    "Should run without failures": {
        "content": [
            [
                "a2112ea5e55a647ce81b27810d0d1070"
            ]
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.10.0"
        },
        "timestamp": "2024-10-28T14:22:04.02598768"
    }
}